KarhooUISDK/Screens/CheckoutScreen/Components/BookingButton/KarhooBookingButtonView.swift

```diff
@@ -94,7 +94,7 @@ final class KarhooBookingButtonView: UIView, BookingButtonView {
     
     override func awakeFromNib() {
         super.awakeFromNib()
-        self.setUpView()
+        setUpView()
```

Comment:
setUpView called twice - in constructor's body and inside 'awakeFromNib'. I assume this is not intended. I am aware you are not the one who created this code, but could you please remove of the calls?